task3.js

```diff
@@ -0,0 +1,13 @@
+// Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную
+
+// n + nn + nnn, где n не перемножаются, а конкатенируются
+
+function getConcateNumber(n) {
+    let str = ''
+    for (let i = 1; i <=n; i++) {
+         str += n * i
+    }
+    return str
+}
+
+console.log(getConcateNumber(5))
```

Comment:
Check the result of your solution

You should get somethig like this
```js
5 + 55 + 555 = 615
```

Consider adding checks for the argument - positive, integer, min: 1, max: 9 value